server_admin/topics/authentication/webauthn.adoc

```diff
@@ -187,3 +186,9 @@ The configurable items and their description follow.
 |The white list of AAGUID of which a WebAuthn authenticator can be registered. This is applied to the operation of registering WebAuthn authenticator. If no entry is set on this list, any WebAuthn authenticator can be registered.
 
 |===
+
+==== Attestation Statement Verification
+
+When registering a webauthn authenticator, {project_name} verifies an attestation statement generated by this webauthn authenticator. On this verification process, {project_name} validates this attestation statement's trustworthiness. It requires trust anchor's certificates. {project_name} uses the link:{installguide_truststore_link}[{installguide_truststore_name}]  so that you need to import these certificates onto it in advance.
```

Comment:
Please use a consistent capitalization.   I believe you are using  WebAuthn  not webauthn

Reply:
I see, I'll use WebAuthn.